utils/mixin.py

```diff
@@ -20,6 +20,8 @@ def get_configuration_template(self):
             return default_config
         elif self.CONFIG_TEMPLATE:
             return self.CONFIG_TEMPLATE
+        else:  # pragma: no cover
```

Comment:
```python
    def get_configuration_template(self):
        default_config = self._default_config
        if default_config:
            return default_config
        else self.CONFIG_TEMPLATE:
            return self.CONFIG_TEMPLATE
        return
```

**?**

Reply:
`else self.CONFIG_TEMPLATE:` is syntax error ;-)

Reply:
oh okay :+1:

Reply:
these were a minor review, discard below comment directly if there will be some error :)